src/cli.rs

```diff
@@ -203,6 +203,11 @@ enum DbCommands {
     Create,
     /// Migrate schema (up)
     Migrate,
+    /// Run one down migration, or add a number to run multiple down migrations (i.e. `down 2`)
+    Down {
+        /// The number of migrations to rollback
+        steps: Option<u32>,
```

Comment:
Please remove the option<u32> and add clippy default as 1.  
this will be clear when users use --help and see what is the default value

Reply:
Is there an example of clippy default?

Reply:
see example https://github.com/loco-rs/loco/blob/master/src/cli.rs#L145